gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/explore.proto

```diff
@@ -30,6 +30,8 @@ message ExploreRequest {
   // by the order by expression.
   int32 limit = 20;
   int32 offset = 21;
+
+  string space_id = 25;
```

Comment:
Are you skipping 22-24 deliberately?

Reply:
It was deliberate but not necessarily with good reason. Some proto authors leave gaps between fields to allow later expansion to be sequenced appropriately - given the previous numbers in this proto, I thought that was what was happening - I can collapse it down.